os/arch/arm/src/armv8-m/mpu.h

```diff
@@ -160,6 +160,11 @@
 #define MPU_RASR_AP_RORO      (6 << MPU_RASR_AP_SHIFT)	/* P:RO   U:RO   */
 #define MPU_RASR_XN             (1 << 28)	/* Bit 28: Instruction access disable */
 
+#ifdef CONFIG_MPU_STACK_OVERFLOW_PROTECTION
+#define MPU_REGION_ADDRESS_ALIGNMENT 32	/* MPU regions must be 32 byte address aligned */
+#define MPU_REGION_MINIMUM_SIZE 32		/* Min MPU region size is 32 bytes */
```

Comment:
Do you want to say `minimum size` or `stack guard size`? If it says minimum size, why is it in `CONFIG_MPU_STACK_OVERFLOW_PROTECTION`?

Reply:
I have added stack guard size as configurable now